lib/src/widgets/device_body.dart

```diff
@@ -87,6 +87,10 @@ class DeviceBody extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     final l10n = AppLocalizations.of(context);
+    final deviceFlags = <String>[
+      for (final flag in device.flags)
+        if (flag.localize(context) != null) flag.localize(context)!
+    ];
```

Comment:
alternatively, we could avoid calling `localize()` twice by using `.whereNotNull()`

Reply:
Yes, this nicer